### PR TITLE
Update README because of Ruby 2.6 incompatibility

### PR DESCRIPTION
**Pull request by dshevtsov**
## This PR is a:

- Maintenance

## Summary

When this pull request is merged, it will:
- ~~remove link checking hook for `jekyll serve`, because Windows does not work with html-proofer on default Ruby installation that, in turn, fails any Jekyll server run.~~ merged in #3560 
- update README.md to consider incompatibility with Ruby 2.6.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,118 +1,153 @@
...
 
-![](https://raw.githubusercontent.com/wiki/magento/magento2/images/dev_docs_contributors.png)
+![DevDocs contributors](https://raw.githubusercontent.com/wiki/magento/magento2/images/dev_docs_contributors.png)
 
-# Building this site
+## Building this site
 
-You can build this site locally in the following ways:
+The website uses [GitHub Pages](https://pages.github.com/) tools for building with the [Jekyll](https://jekyllrb.com/) static website generator on top of it.
+The toolkit requires a Ruby environment to be run.
```
Comment (hguthrie) [inline README.md]: remove "be"

File: README.md [hunk 2]
```diff
@@ -1,118 +1,153 @@
...
 
-- [Installing the project dependencies locally](#build-using-jekyll) (Mac, Linux)
+There are several options available depending on what type and level of virtualization you want to use:
+
+- [Installing the project dependencies locally](#build-using-jekyll)
 - [Using a Docker container](#build-using-docker) (Mac, Linux)
-- [Using a Vagrant virtual machine](#build-using-vagrant) (Mac, Linux, Windows)
-- [Build DevDocs in Windows](#build-devdocs-in-windows) (Windows 7 & 10)
+- [Using a Vagrant virtual machine](#build-using-vagrant)
```
Comment (hguthrie) [inline README.md]: Why did you remove the link for Windows? I tested that section thoroughly, as did the Cloud team. It was a request from both inside and outside users.

File: README.md [hunk 3]
```diff
@@ -1,118 +1,153 @@
...
 
-## Build using Jekyll
+## Installing the project dependencies locally
```
Comment (hguthrie) [inline README.md]: Do not say "locally"

You don't need this title and Prerequisites. Choose one.

File: README.md [hunk 4]
```diff
@@ -1,118 +1,153 @@
...
+                    done in 14.865 seconds.
+Auto-regeneration: enabled for '/Users/user/Projects/repos/magento/devdocs'
+LiveReload address: http://127.0.0.1:35729
+    Server address: http://127.0.0.1:4000/
+Server running... press ctrl-c to stop.
+        LiveReload: Browser connected
+```
 
-1. Press `Ctrl+C` in the serve terminal to stop the server.
+This command:
```
Comment (hguthrie) [inline README.md]: This is a strange way to introduce the following. It would be better to say 
"The 'rake' task performs the following actions:"
Or something similar to remind them what command you are describing.

File: README.md [hunk 5]
```diff
@@ -134,18 +169,14 @@ The following example will generate Magento 2.2 documentation only.
...
-1. Run the preview command:
-
-   ```
-   $ rake preview
-   ```
-   This command:
-   * Checks your environment according to the dependencies in `Gemfile.lock`.
-   * Removes the `_site/` directory, which contains previously generated preview files.
-   * Generates a new preview and opens the landing page in a web browsers.
+2. Run [`rake preview`](#using-rake)
```
Comment (hguthrie) [inline README.md]: The step should say what you are doing.
"Use the following rake command to generate a preview in a web browser."

File: README.md [hunk 6]
```diff
@@ -187,133 +218,60 @@ You can deploy the devdocs site locally using [this Vagrant project](https://git
...
-*	<a href="https://devdocs.magento.com">Visit our documentation site</a>, built using [GitHub pages](https://pages.github.com/).
+- [Follow @MagentoDevDocs](https://twitter.com/MagentoDevDocs)
+- [E-mail us](mailto:DL-Magento-Doc-Feedback@magento.com)
+- [Visit our documentation site](https://devdocs.magento.com), built using [GitHub pages](https://pages.github.com/).
 
-## Build DevDocs in Windows
+## Build on Windows
 
-Some of the technologies we use to develop DevDocs is not compatible with Windows, such as [Jekyll](https://jekyllrb.com/docs/windows/). For this reason, we do not support DevDocs management in Windows; however, we have documented the following procedures to build the DevDocs in a Windows environment. Any further use of this setup or troubleshooting is up to you. 
+Windows does not provide the out-of-the-box Ruby, so we recommend to use a virtual environment with Docker or Vagrant described above.
```
Comment (hguthrie) [inline README.md]: Do we? I don't know. I had my Cloud windows users look at this before and I disagree. I did a lot of research into the Windows method I documented.

File: README.md [hunk 7]
```diff
@@ -1,118 +1,153 @@
...
 Our goal is to provide the Magento community with comprehensive and quality technical documentation. We believe that to accomplish that goal we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving Magento documentation.
 
-![](https://raw.githubusercontent.com/wiki/magento/magento2/images/dev_docs_contributors.png)
+![DevDocs contributors](https://raw.githubusercontent.com/wiki/magento/magento2/images/dev_docs_contributors.png)
 
-# Building this site
+## Building this site
 
-You can build this site locally in the following ways:
+The website uses [GitHub Pages](https://pages.github.com/) tools for building with the [Jekyll](https://jekyllrb.com/) static website generator on top of it.
```
[the remark this answers is not in the document]
Comment (dshevtsov) [reply README.md]: We use the github-pages gem as a our main gem: https://pages.github.com/versions/
We can consider moving to clean Jekyll direction to be able to manage the dependencies manually and use the latest version of Jekyll.

[the remark this answers is not in the document]
Comment (jeff-matthews) [reply README.md]: I like the idea of having more control over our deps.

**Comment by dshevtsov** [comment]
@hguthrie feel free to modify the PR to meet your requirements and expectations

**Comment by hguthrie** [comment]
Readme update suggestions in review #3593

**Comment by hguthrie** [comment]
@jeff-matthews  @dshevtsov  What is up with this PR? Now the link checking hook is no longer a part of this? I don't understand.

**Comment by dshevtsov** [comment]
@hguthrie yes, the hook update was merged separately in #3560

**Comment by hguthrie** [comment]
This is the PR you speak of. It has not been merged. How is it separate?

**Comment by dshevtsov** [comment]
wrong link, sorry
that's the PR #3618

**Comment by dshevtsov** [comment]
devdocs builds fine with Ruby 2.6. after #3753
